test/parallel/test-fs-read-buffer-tostring-fail.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 
 const common = require('../common');
+
+const skipMessage = 'intensive toString tests due to memory confinements';
```

Comment:
I'm not sure what you mean. Here's another example: https://github.com/nodejs/node/blob/b10ee9deaee90bb7d6f9ef938ebcc94c0380969f/test/addons/stringbytes-external-exceed-max/test-stringbytes-external-at-max.js#L11-L15

And here's what this test pass when it skips:

```
1..0 # Skipped: intensive toString tests due to memory confinements
```